src/vse_sync_pp/analyzers/ppsdpll.py

```diff
@@ -2,29 +2,16 @@
 
 """Analyze ppsdpll log messages"""
 
-from .analyzer import Analyzer
+from .analyzer import TimeErrorAnalyzer
 
-class PhaseOffsetTimeErrorAnalyzer(Analyzer):
-    """Analyze DPLL phase offset time error"""
-    id_ = 'ppsdpll/phase-offset-time-error'
-    parser = 'dpll/phase-offset'
+class PpsDpllAnalyzer(TimeErrorAnalyzer):
```

Comment:
```suggestion
class TimeErrorAnalyzer(TimeErrorAnalyzerBase):
```

The name of the class should reflect the `id_`. I suspect you chose a different name to avoid a collision with the base?

Reply:
yes, I like more TimeErrorAnalyzerBase you are proposing.

Reply:
+1 Adding the base suffix makes it clear it shouldn't be used for actual analysis